README.md

```diff
@@ -58,6 +58,44 @@ spec:
             - Resource
 ```
 
+
+When adding a new policy, please follow these guidelines:
+
+1. Create a new folder with the name of the policy.
+2. Add the Kyverno policy YAML file to the folder.
+3. Add an `artifacthub-pkg.yml` metadata file to the folder. See the [Template Policy Metadata File](https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml) section for an example and instructions on how to complete the contents of this file.
+
+```yaml
+# artifacthub-pkg.yml
+version: '1.0.0'
+name: your_policy_name
+displayName: Your Policy Display Name
+createdAt: "2023-03-29T00:00:00.000Z"
+description: Brief description of your policy
+author: Your Name
+digest: Result of `shasum -a 256 artifacthub-pkg.yml`
+install: |-
+    ```shell
+    kubectl apply -f certain_policy.yaml
+     ```
+license: Apache-2.0
+keywords:
+  - policy
+  - example
+  - security
+readme: |
+  Copy the content of you policy description here.
+  
+  Refer to the documentation for more details on Kyverno annotations: https://artifacthub.io/docs/topics/annotations/kyverno/
+
+annotations:
+  kyverno/category: refer to policies.kyverno.io/category in policy file 
+  kyverno/kubernetesVersion: refer to kyverno.io/kubernetes-version in policy file
+  kyverno/subject: refer to policies.kyverno.io/subject in policy file
+  
+```
```

Comment:
Please replace all this with one of the examples you created from the Velero folder. In addition, add to the end of each line a comment which describes what goes in each value.